docker-compose.yml

```diff
@@ -56,9 +56,58 @@ services:
       - "traefik.enable=true"
       - "traefik.docker.network=frontend"
       - "traefik.http.routers.${COMPOSE_PROJECT_NAME}.rule=Host(`${COMPOSE_DOMAIN}`)"
-#      HTTPS config - uncomment to enable redirect from :80 to :443
-#      - "traefik.http.routers.${COMPOSE_PROJECT_NAME}.middlewares=redirect-to-https"
-#      - "traefik.http.middlewares.redirect-to-https.redirectscheme.scheme=https"
+  #      HTTPS config - uncomment to enable redirect from :80 to :443
+      - "traefik.http.routers.${COMPOSE_PROJECT_NAME}.middlewares=redirect-to-https"
+      - "traefik.http.middlewares.redirect-to-https.redirectscheme.scheme=https"
+
+  engine:
+    image: node:14
```

Comment:
Should we upgrade to `node:16` or `node:18` while we are at it? 
`14`: Maintenance LTS, EOL 2023-04-30
`16`: Maintenance LTS, EOL 2023-09-11
`18`: Active LTS

Reply:
Upgraded to node 18. Will require that the server setup is updated to node 18 as well.

Reply:
Nice catch. Forgot we are not on a docker setup on the server. Let's leave it at `14` now. But we need a task for upgrading soon.

Reply:
Downgraded to node 14 again